src/structures/Guild.js

```diff
@@ -866,6 +875,7 @@ class Guild extends AnonymousGuild {
       _data.description = data.description;
     }
     if (data.preferredLocale) _data.preferred_locale = data.preferredLocale;
+    if (data.premiumProgressBarEnabled) _data.premium_progress_bar_enabled = data.premiumProgressBarEnabled;
```

Comment:
This won't set it to false if i call it with false...

Reply:
right, my bad
```suggestion
    if ('premiumProgressBarEnabled' in data) _data.premium_progress_bar_enabled = data.premiumProgressBarEnabled;
```